spec/java/util/_interfaces.lsl

```diff
@@ -19,78 +23,143 @@ type Comparator
 }
 
 
-@Parameterized(["E"]) // #problem
+@Parameterized(["E"])
 type Iterator
     is java.util.Iterator
     for Object
 {
+    fun hasNext(): boolean;
+
+    fun next(): Object;
 }
 
 
-@Parameterized(["T"]) // #problem
+@Parameterized(["T"])
 type Spliterator
     is java.util.Spliterator
     for Object
 {
 }
 
 
-@Parameterized(["K", "V"]) // #problem
+@Parameterized(["K", "V"])
 type Map
```

Comment:
As I see there are more methods in [java.util.Map](https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/util/Map.html)
For example: "replace"; They weren't added because it is impossible now ?